var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -250,6 +250,6 @@ def write_rpath_specs(self):
             for line in lines:
                 out.write(line + '\n')
                 if line.startswith('*link:'):
-                    out.write(r'-rpath {0}:{1} \n'.format(
```

Comment:
I'm used to Expect, where you actually _do_ have to triply escape new line characters.